### PR TITLE
Issue 351: Added Reset button 

### PR DESCRIPTION
**Pull request by christinem**
This PR fixes Issue #351, and adds the reset button back, in the sidebar.

This PR is built on #513.

### CODE REVIEW

**Comment by david-yz-liu** [comment]
1. It should be immediately underneath FCE count, and appear regardless of which tab is selected
2. Give it the label "reset graphs"

**Comment by christinem** [comment]
@david-yz-liu I made the changes (the commit is slightly misnamed), however putting the button directly under the FCE count made it look a little awkward, so I put it beside the FCE count, and is always visible - let me know if you think that is alright.

File: hs/GraphResponse.hs [hunk 1]
```diff
@@ -52,7 +52,9 @@ timetableTab =
 sideBar :: H.Html
 sideBar = do
     H.div ! A.id "sidebar" $ do
-        H.div ! A.id "fcecount" $ ""
+        H.div ! A.id "fce" $ do
+            H.div ! A.id "fcecount" $ ""
+            H.button ! A.id "reset" $ "Reset Graphs"
```
Comment (Ian-Stewart-Binks) [inline hs/GraphResponse.hs]: No money (`$`) necessary. `$` is *similar* to using parentheses (however, *not* identical).

[Here's a good source on the matter](http://learnyouahaskell.com/higher-order-functions)
Scroll down to *"Function application with $"*.

**Comment by david-yz-liu** [comment]
@christinem I merged this into master.

**Comment by christinem** [comment]
@david-yz-liu thanks!

**Pull request closed without merging**